packages/p2p/src/stores/general-store.js

```diff
@@ -386,6 +387,15 @@ export default class GeneralStore extends BaseStore {
         });
     }
 
+    showDailyLimitIncreaseNotification() {
```

Comment:
**suggestion:** shouldn't showDailyLimitIncreaseNotification be added above as `showDailyLimitIncreaseNotification: action.bound`? because I didn't see it...